shuup/admin/forms/fields.py

```diff
@@ -37,6 +37,19 @@ def widget_attrs(self, widget):
         return attrs
 
 
+class DecimalPlaceField(DecimalField):
+    def widget_attrs(self, widget):
+        attrs = super().widget_attrs(widget)
+        if self.min_value is not None:
+            attrs["min"] = self.min_value
+        if self.max_value is not None:
+            attrs["max"] = self.max_value
+        if self.decimal_places == 0:
+            attrs["onkeydown"] = "if(event.key==='.'){event.preventDefault();}"
```

Comment:
this won't work for different locations, for example here in Brazil the decimal separator is `,`. Can't you control this through the input `step='0.01'` property instead?